Octokit/Models/Request/SearchRepositoriesRequest.cs

```diff
@@ -479,9 +479,9 @@ public enum Language
         [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Edn")]
         Edn,
         Eiffel,
-        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Elixer")]
-        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Elixer")]
-        Elixer,
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Elixir")]
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Elixir")]
+        Elixir,
```

Comment:
I have one remarks about it. Yes, it is breaking changes, but this fully incorrect items (**Elixer**, Unified **Paralel ** C), these items never works correctly and if somebody use these then he doesn't  get any search results. Anyway, I reverted older items back and added new correct items. Hope they will be removed in further release.